package/tests/test_integrations/test_telemetry.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2021 QuantumBlack Visual Analytics Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+# EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES
+# OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE, AND
+# NONINFRINGEMENT. IN NO EVENT WILL THE LICENSOR OR OTHER CONTRIBUTORS
+# BE LIABLE FOR ANY CLAIM, DAMAGES, OR OTHER LIABILITY, WHETHER IN AN
+# ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF, OR IN
+# CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+#
+# The QuantumBlack Visual Analytics Limited ("QuantumBlack") name and logo
+# (either separately or in combination, "QuantumBlack Trademarks") are
+# trademarks of QuantumBlack. The License does not grant you any right or
+# license to the QuantumBlack Trademarks. You may not use the QuantumBlack
+# Trademarks or any confusingly similar mark as a trademark for your product,
+# or use the QuantumBlack Trademarks in any other manner that might cause
+# confusion in the marketplace, including but not limited to in advertising,
+# on websites, or on software.
+#
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from pathlib import Path
+from unittest import mock
+
+from kedro_viz.integrations.kedro import telemetry as kedro_telemetry
+
+
+def test_get_heap_app_id_no_telemetry_file():
+    assert kedro_telemetry.get_heap_app_id(Path.cwd()) is None
+
+
+def test_get_heap_app_id_invalid_telemetry_file(tmpdir):
+    telemetry_file = tmpdir / ".telemetry"
+    telemetry_file.write_text("foo", encoding="utf-8")
+    assert kedro_telemetry.get_heap_app_id(tmpdir) is None
+
+
+def test_get_heap_app_id_no_consent(tmpdir):
+    telemetry_file = tmpdir / ".telemetry"
+    telemetry_file.write_text("consent: false", encoding="utf-8")
+    assert kedro_telemetry.get_heap_app_id(tmpdir) is None
+
+
+@mock.patch("kedro_viz.integrations.kedro.telemetry._get_heap_app_id")
```

Comment:
There's a mix of `unittest` and `pytest` in the `tests` folder. 😂  Assuming this still runs when calling `pytest` right?